binancial/data/get_klines_realtime.py

```
@@ -48,30 +71,57 @@ def handle_socket_message(msg):
-                              symbol=symbol,
-                              interval='1m')
+            logger.error(f"Error processing message: {error}")
+            logger.error(f"Message content: {msg}")
+            if socket_id:
+                stream.stop_socket(socket_id)
+            raise
+    
+    try:
+        # Open file with context manager
+        with open(file_path, 'a') as file:
+            logger.info(f"Starting kline stream for {symbol} with interval {interval}")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)